unblob/handlers/compression/lzma.py

```diff
@@ -17,7 +17,14 @@ class LZMAHandler(Handler):
         strings:
             $lzma_magic = { 5d 00 00 ( 00 | 01 | 04 | 08 | 10 | 20 | 40 | 80) ( 00 | 01 | 02 | 04 | 08 ) }
         condition:
-            $lzma_magic
+            $lzma_magic and
```

Comment:
It is easier to figure out then in the case of unix compress where can I look into the spec but it would be nice if it was referred here as well.

Reply:
Added LZMA file format reference and link to specific sections of that reference for each validation we do.